etc/config/cflat.defaults.properties

```diff
@@ -0,0 +1,4 @@
+compilers=cfalt
+compilerType=cflat
+compiler.cflat.name=cflat
+compiler.cflat.exe=/Users/arbie/Documents/CS590/farrago-master/target/debug/lirc
```

Comment:
Your particular executable path shouldn't be committed to the repo. You can have it in your local properties file.

Reply:
What should the execution route of compiler.cflat.exe be for cflat.defaults.properties?

Reply:
It can be just `lirc` for now, or if not having that property works, you can just remove it.